### ISSUE TITLE
Error running baysor preview - raises MethodError.

### ISSUE THREAD

**Issue by alipirani88** [kharchenkolab/Baysor#138]
How do I solve this error? Would appreciate any help solving this issue?

ERROR: MethodError: no method matching tryparse(::Type{Symbol}, ::String)

Closest candidates are:
  tryparse(::Type{Union{}}, ::Any...; kwargs...)
   @ Base parse.jl:257
  tryparse(::Type{FilePathsBase.WindowsPath}, ::AbstractString)
   @ FilePathsBase /home/viktor_petukhov/.julia/packages/FilePathsBase/Laclt/src/windows.jl:31
  tryparse(::Type{Float16}, ::AbstractString)
   @ Base parse.jl:302
  ...

Stacktrace:
 [1] command_main(ARGS::Vector{String})
   @ Baysor /home/viktor_petukhov/.julia/packages/Comonicon/F3QqZ/src/codegen/julia.jl:490
 [2] command_main
   @ /home/viktor_petukhov/.julia/packages/Comonicon/F3QqZ/src/codegen/julia.jl:90 [inlined]
 [3] julia_main()
   @ Baysor /home/viktor_petukhov/.julia/packages/Comonicon/F3QqZ/src/frontend/cast.jl:481
 [4] top-level scope
   @ none:1

**Comment by alipirani88** [comment]
Hi @VPetukhov ,

Would appreciate any help in resolving this issue.

Thanks,

**Comment by VPetukhov** [comment]
Please, provide more information. The OS, the command you ran and full error stacktrace and full log.

**Comment by alipirani88** [comment]
I was able to resolve this by downgrading to v0.6.2 and using binaries. It seems the issue is specific to the latest version.

Operating System: Red Hat Enterprise Linux 8.7 (Ootpa)
CPE OS Name: cpe:/o:redhat:enterprise_linux:8::baseos
Kernel: Linux 4.18.0-425.3.1.el8.x86_64
Architecture: x86-64

Commands:

`baysor run -x x_location -y y_location -z z_location -g feature_name -m 5 -p -s 5 -o /rsrch5/home/neuro_rsrch/apirani/20240608__175715__Matri_GliomaProTMA_Met_SS_060824/baysor_run/default/ ~/20240608__175715__Matri_GliomaProTMA_Met_SS_060824/X0.0-24000.0_Y0.0-24000.0_filtered_transcripts_changed_cellid.csv`

By the way, I am still getting the same error as my earlier post when I try to build it in MacOS. I was not able to build it in either MAC or Linux so had to opt for the binaries.

**Comment by alipirani88** [comment]
I was able to resolve this by downgrading to v0.6.2 and using binaries. It seems the issue is specific to the latest version.

Operating System: Red Hat Enterprise Linux 8.7 (Ootpa)
CPE OS Name: cpe:/o:redhat:enterprise_linux:8::baseos
Kernel: Linux 4.18.0-425.3.1.el8.x86_64
Architecture: x86-64

Commands:

baysor run -x x_location -y y_location -z z_location -g feature_name -m 5 -p -s 5 -o /rsrch5/home/neuro_rsrch/apirani/20240608__175715__Matri_GliomaProTMA_Met_SS_060824/baysor_run/default/ ~/20240608__175715__Matri_GliomaProTMA_Met_SS_060824/X0.0-24000.0_Y0.0-24000.0_filtered_transcripts_changed_cellid.csv

By the way, I am still getting the same error as my earlier post when I try to build it in MacOS. I was not able to build it in either MAC or Linux so had to opt for the binaries.

**Comment by marzaidi** [comment]
hi, i had the same initial issue as you so i tried out your method with v0.6.2 and it installed successfully. however, every time i run the command, it takes a long time and eventually throws this error: 

[11:53:05] Info: Run R61a6fa158
[11:53:06] Info: (2024-09-28) Run Baysor v0.6.2
[11:53:06] Info: Loading data...
[11:53:34] Info: Loaded 11908031 transcripts
[11:53:39] Info: Estimating noise level
[11:59:46] Info: Done
[12:02:51] Info: Clustering molecules...
Progress: 100%|█████████████████████████████████████████████████████████████████| Time: 0:27:00        
  Iteration:                  469
  Max. difference:            0.00642
  Fraction of probs changed:  0.00171
[12:34:31] Info: Algorithm stopped after 469 iterations. Error: 0.00642. Converged: true.
[12:34:32] Info: Done
[12:34:48] Info: Initializing algorithm. Scale: 5.0, scale std: 1.25, initial #components: 4763212, #molecules: 11908031.
[12:48:07] Info: Using the following additional information about molecules: [:confidence, :cluster]
[12:48:07] Info: Using 3D coordinates
Killedss:   4%|██▍                                                              |  ETA: 7 days, 0:08:01 

i even attempted to use a tmux session. did you have a similar issue?

**Comment by alipirani88** [comment]
I didnt come across this error. However, I ran the jon on a high memory node (~350Gb). If you can post the exact error, that might tell us what the actual issue is.

**Comment by alipirani88** [comment]
@marzaidi Yes - still not able to use the latest version due to build errors and the binaries dont work out of the box. @VPetukhov  Hopefully, there will be a fix soon.

**Comment by VPetukhov** [comment]
@marzaidi 

> Killedss: 4%|██▍ | ETA: 7 days, 0:08:01

This suggests that the job was killed, which usually happens because too much memory was used. Baysor is [memory-hungry](https://github.com/kharchenkolab/Baysor/issues/87), and 10x datasets are really big.

https://github.com/gustaveroussy/sopa may help here.

**Comment by VPetukhov** [comment]
@alipirani88 , could you please submit build errors to a separate issues?

**Comment by VPetukhov** [comment]
By the way, quick fix, try using [config](https://github.com/kharchenkolab/Baysor/blob/master/configs/xenium.toml) (`-c` option) instead of passing parameters manually.

**Comment by melchuawy** [comment]
Hello,

The error still persists even with the latest version.

**Comment by VPetukhov** [comment]
Hello @melchuawy , do you mean v0.7.0 or `develop`?

**Comment by VPetukhov** [comment]
This is fixed in v0.7.1.
